site/assets/scss/_content.scss

```diff
@@ -140,15 +140,13 @@
 
   .dropdown-menu {
     --bs-dropdown-bg: #{mix($blue-500, $blue-600)};
-    --bs-dropdown-link-color: var(--bs-white);
     --bs-dropdown-link-active-color: var(--bs-white);
     --bs-dropdown-link-active-bg: #{$blue-700};
     --bs-dropdown-link-hover-color: var(--bs-white);
```

Comment:
This one was added by #2030.

Reply:
Yes, but it doesn't seem to be useful, isn't it?

Reply:
I'm going to test the other variables as well. Good catch!